### PR TITLE
Update homepage functional test for newsletter submission

### PR DESCRIPTION
**Pull request by alexgibson**

### CODE REVIEW

**Comment by alexgibson** [comment]
@schalkneethling r? I got a couple of intermittent failures for this test yesterday, due to either slow network connectivity or sendto.mozilla.org being slow. This should ease things a bit.

File: tests/lib/config.js [hunk 1]
```diff
@@ -28,6 +28,9 @@ var userAgent = {
 // set default viewport size to desktop
 casper.options.viewportSize = viewport.desktop;
 
+// set default wait timeout
+casper.options.waitTimeout = 5000;
```
Comment (schalkneethling) [inline tests/lib/config.js]: Is resetting it in helpers#done not enough? The default timeout in Casperjs is 5000

Comment (alexgibson) [inline tests/lib/config.js]: Figured it couldn't hurt, I guess the default Casper uses could always change in future versions?

Comment (schalkneethling) [inline tests/lib/config.js]: Fair enough :+1:

**Comment by schalkneethling** [comment]
r+

**Pull request merged**